Reference/Management/Models/Template-v9.md

```diff
@@ -0,0 +1,93 @@
+---
+versionFrom: 9.0.0
+verified-against: rc001
+meta-title: Template Model
+meta.Description: Represents a Template file.
```

Comment:
```suggestion
meta.Description: "Represents a Template file."
```

Reply:
@busrasengul Updated with you suggestions